applications/FluidDynamicsApplication/tests/cpp_tests/test_stokes_two_fluid.cpp

```diff
@@ -0,0 +1,236 @@
+                for (unsigned int k = 0; k < 3; k++) {
+                    pElement->GetGeometry()[i].FastGetSolutionStepValue(VELOCITY)[k] = vel_original(i, k);
+                    pElement->GetGeometry()[i].FastGetSolutionStepValue(VELOCITY, 1)[k] = 0.9*vel_original(i, k);
+                    pElement->GetGeometry()[i].FastGetSolutionStepValue(VELOCITY, 2)[k] = 0.75*vel_original(i, k);
+                    pElement->GetGeometry()[i].FastGetSolutionStepValue(MESH_VELOCITY)[k] = 0.0;
+                    pElement->GetGeometry()[i].FastGetSolutionStepValue(MESH_VELOCITY, 1)[k] = 0.0;
+                    pElement->GetGeometry()[i].FastGetSolutionStepValue(MESH_VELOCITY, 2)[k] = 0.0;
+                }
+            }
+        }
+
+        // /** Checks the StokesTwoFluid3D4N element
+        //  * Checks the LHS and RHS for a cut element
+        //  */
+        KRATOS_TEST_CASE_IN_SUITE(ElementStokesTwoFluidCut3D4N, FluidDynamicsApplicationFastSuite)
+        {
+            Model current_model;
+            ModelPart& modelPart = current_model.CreateModelPart("Main");
+            PrepareModelPart(modelPart);
+            
+            auto pElement = modelPart.pGetElement(1);
+
+            pElement->GetGeometry()[0].FastGetSolutionStepValue(DISTANCE) = -1.0;
+            pElement->GetGeometry()[1].FastGetSolutionStepValue(DISTANCE) =  1.0;
+            pElement->GetGeometry()[2].FastGetSolutionStepValue(DISTANCE) = -1.0;
+            pElement->GetGeometry()[3].FastGetSolutionStepValue(DISTANCE) =  1.0;
+
+            // Compute RHS and LHS
+            Vector RHS = ZeroVector(16);
+            Matrix LHS = ZeroMatrix(16,16);
+
+            const auto& r_process_info = modelPart.GetProcessInfo();
+            pElement->Initialize(r_process_info); // Initialize the element to initialize the constitutive law
+            pElement->CalculateLocalSystem(LHS, RHS, r_process_info);
+
+            // Check the RHS values (the RHS is computed as the LHS x previous_solution,
+            // hence, it is assumed that if the RHS is correct, the LHS is correct as well)
+            KRATOS_CHECK_NEAR(RHS(0), -8.86905, 1e-2);
+            KRATOS_CHECK_NEAR(RHS(1), -0.37064, 1e-2);
+            KRATOS_CHECK_NEAR(RHS(2), 7.71903, 1e-2);
+            KRATOS_CHECK_NEAR(RHS(3), -318569, 1.0);
+            KRATOS_CHECK_NEAR(RHS(4), 22.639, 1e-2);
+            KRATOS_CHECK_NEAR(RHS(5), 8.65338, 1e-2);
+            KRATOS_CHECK_NEAR(RHS(6), 12.1078, 1e-2);
+            KRATOS_CHECK_NEAR(RHS(7), 2866.56, 1e-2);
+            KRATOS_CHECK_NEAR(RHS(8), 12.9595, 1e-2);
+            KRATOS_CHECK_NEAR(RHS(9), 39.2958, 1e-2);
+            KRATOS_CHECK_NEAR(RHS(10), 29.5256, 1e-2);
+            KRATOS_CHECK_NEAR(RHS(11), 316699, 1.0);
+            KRATOS_CHECK_NEAR(RHS(12), 5.71452, 1e-2);
+            KRATOS_CHECK_NEAR(RHS(13), 9.57769, 1e-2);
+            KRATOS_CHECK_NEAR(RHS(14), 30.8809, 1e-2);
+            KRATOS_CHECK_NEAR(RHS(15), -996.765, 1e-2);
```

Comment:
This increases a lot compilation time. Please use `KRATOS_CHECK_VECTOR_NEAR` instead.